scripts/validate_docstrings.py

```diff
@@ -691,7 +691,10 @@ def get_validation_data(doc):
 
     if doc.is_function_or_method:
         if not doc.returns:
-            if 'return' in doc.method_source:
+            if re.search('\"\"\".*\"\"\".*' +
```

Comment:
This is rather difficult to reason about; can you use `re.VERBOSE` here and add comments?

Reply:
Hey @WillAyd, I have added some comments, but am not sure if they are enough. Can you give me some more feedback?